src/api/api.ts

```diff
@@ -433,6 +433,10 @@ export const axiosRegisterTutor = (tutor: Tutor) => {
 };
 
 export const axiosCreateCourse = (token: string, course: Course) => {
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  // eslint-disable-next-line no-param-reassign
+  course.allowContact = false;
```

Comment:
Why is `allowContact` set `false` here?

Reply:
Oops, good catch 😅 I added this because the branch I implemented changes on was a bit old and therefore didn't work with the current backend anymore. Removed it in a commit, thanks for pointing this out! 👍